src/m365/spo/commands/list/list-roleinheritance-reset.ts

```diff
@@ -0,0 +1,94 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+}
+
+class SpoListRoleInHeritanceResetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LIST_ROLEINHERITANCE_RESET;
+  }
+
+  public get description(): string {
+    return 'Restore role inheritance on list or library.';
+  }
+
+  public optionSets(): string[][] | undefined {
+    return [
+      [ 'listId', 'listTitle' ]
+    ];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`Reset role inheritance of list in site at ${args.options.webUrl}...`);
```

Comment:
Since we're referring to it as `restore` in the command's description, let's use the same word here instead of `reset` for consistency